docs/table-of-contents.json

```diff
@@ -4,50 +4,16 @@
     {
       "title": "Overview",
       "entries": [
-        {
-          "entry": "docs/README"
-        },
-        {
-          "entry": "docs/whats-new"
-        },
-        {
-          "entry": "docs/roadmap"
-        },
-        {
-          "entry": "docs/FAQ"
-        },
-        {
-          "entry": "docs/upgrade-guide"
-        }
+        { "entry": "docs/README" },
+        { "entry": "docs/whats-new" },
+        { "entry": "docs/upgrade-guide" }
       ]
     },
     {
       "title": "Getting Started",
       "entries": [
-        {
-          "name": "Overview",
-          "entry": "docs/get-started/README"
-        },
-        {
-          "name": "Installation",
-          "entry": "docs/get-started/installation"
-        },
-        {
-          "name": "Examples",
-          "entry": "docs/get-started/examples"
-        },
-        {
-          "name": "Using with deck.gl",
-          "entry": "docs/get-started/using-with-deckgl"
-        },
-        {
-          "name": "Using with Node.js",
-          "entry": "docs/get-started/using-with-node"
-        },
-        {
-          "name": "Using with other Frameworks",
-          "entry": "docs/get-started/using-with-other-frameworks"
-        }
+        { "entry": "docs/get-started/README" },
```

Comment:
Just a note: In other repos I am moving these into `developer-guide/get-started`...